explainer.md

```diff
@@ -0,0 +1,66 @@
+# Extension document explainer
+This document is a brief explainer for new features added in the [extension document](https://w3c.github.io/pointerevents/extension.html).
+
+## Coalesced Events
+Nowadays there are high frequency devices that generate user inputs much faster than frame refresh rate in the apps.
+Hence some browsers chose to not deliver all those events to the web pages as they realized it is going to be a lot of extra work that doesn't necessary have an effect on the visual output.
+For example Chromium delays all move events until right before rAF and Firefox does the same if the first event in the frame causes any changes to the DOM.
+
+### Problem
+Since most of the web apps care only about the latest state of the input before generating the next frame, the events happened during the frame could be coalesced together into one event that gets dispatched to the page.
+In this case the dispatched event could have the latest screenX/Y coordinates and its movementX/Y could be the sum of movementX/Y of all the events that got coalesced into the dispatch event.
+This greatly helped reduce the amount of unnecessary work by UA for routing, hittesting and such for most of the web apps.
+However, there is small set of apps like drawing apps that might still care about the historical details of all those events.
+For instance they need to know the exact path of the pointer to be able to draw a smooth line.
+So although coalescing events and dispatching the coalesced event once per frame would greatly help in terms of performance we still need an API on the dipatched event to expose the coalesced events information in case the app wants to take advantage of that.
+
+### Solution
+We propose to provide an API called `getCoalescedEvents` on `PointerEvent` object so that it exposes any events that have been coalesced into the dispatched event.
+This API also matches other platforms such [getHistoricalX/Y](https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int,%20int)) on Android and [similar API](https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/handling_touches_in_your_view/getting_high-fidelity_input_with_coalesced_touches) on iOS.
+There are two options for exposing the coalesced events:
+1. Return a list of dictionary of interesting and useful attributes such as cooridnates.
+2. Return a list of `PointerEvent` object representing the actual events that have been coalesced.
+
+Each item in the result list corresponds to one coalesced event.
+The first option has the benefit of not having an actual `Event` object and hence there is no need to deal with set of functionalities that we don't need them on coaleaced event suchs as propagation functions in the specification.
```

Comment:
"no need to deal with a set of functionalities that we don't need on coalesced events, such as..."